diem-move/diem-framework/DPN/releases/artifacts/current/build/DiemCoreFramework/sources/SystemAdministrationScripts.move

```diff
@@ -160,4 +161,62 @@ module DiemFramework::SystemAdministrationScripts {
+    /// | `account`       | `signer`      | Signer of the sending account. Must be the Diem Root account.              |
+    /// | `sliding_nonce` | `u64`         | The `sliding_nonce` (see: `SlidingNonce`) to be used for this transaction. |
+    /// | `config`        | `vector<u8>`  | The serialized bytes of ReadWriteSet analysis result.                                  |
+    ///
+    /// # Common Abort Conditions
+    /// | Error Category             | Error Reason                                  | Description                                                                                |
+    /// | ----------------           | --------------                                | -------------                                                                              |
+    /// | `Errors::NOT_PUBLISHED`    | `SlidingNonce::ESLIDING_NONCE`                | A `SlidingNonce` resource is not published under `account`.                                |
+    /// | `Errors::INVALID_ARGUMENT` | `SlidingNonce::ENONCE_TOO_OLD`                | The `sliding_nonce` is too old and it's impossible to determine if it's duplicated or not. |
+    /// | `Errors::INVALID_ARGUMENT` | `SlidingNonce::ENONCE_TOO_NEW`                | The `sliding_nonce` is too far in the future.                                              |
+    /// | `Errors::INVALID_ARGUMENT` | `SlidingNonce::ENONCE_ALREADY_RECORDED`       | The `sliding_nonce` has been previously recorded.                                          |
+    /// | `Errors::REQUIRES_ADDRESS` | `CoreAddresses::EDIEM_ROOT`                   | `account` is not the Diem Root account.                                                    |
+
+    public(script) fun set_parallel_execution(account: signer, sliding_nonce: u64, config: vector<u8>) {
+        SlidingNonce::record_nonce_or_abort(&account, sliding_nonce);
+        ParallelExecutionConfig::set_parallel_execution(&account, config);
+    }
+
+    ///  # Summary
+    /// Updates the ParallelExecutionConfig that is stored on-chain to disable parallel execution.  This
+    /// transaction can only be sent from the Diem Root account.
+    ///
+    /// # Technical Description
+    /// Updates the `ParallelExecutionConfig` on-chain config and emits a `DiemConfig::NewEpochEvent` to trigger
+    /// a reconfiguration of the system.
+    ///
+    /// # Parameters
+    /// | Name            | Type          | Description                                                                |
+    /// | ------          | ------        | -------------                                                              |
+    /// | `account`       | `signer`      | Signer of the sending account. Must be the Diem Root account.              |
+    /// | `sliding_nonce` | `u64`         | The `sliding_nonce` (see: `SlidingNonce`) to be used for this transaction. |
+    /// | `config`        | `vector<u8>`  | The serialized bytes of ReadWriteSet analysis result.                                  |
```

Comment:
The script doesn't have this argument :)

Reply:
I should probably just get rid of the scripts here as they will be used by admin scripts.